### PR TITLE
Add transitive reference to PlatformAbstractions

### PR DESCRIPTION
**Pull request by ryanbrandenburg**

### CODE REVIEW

File: src/Microsoft.Extensions.CodeGeneration.Utils/project.json [hunk 1]
```diff
@@ -5,7 +5,8 @@
   "version": "1.0.0-*",
   "dependencies": {
     "Microsoft.CodeAnalysis.CSharp.Workspaces": "1.3.0-*",
-    "Microsoft.DotNet.ProjectModel": "1.0.0-*"
+    "Microsoft.DotNet.ProjectModel": "1.0.0-*",
+    "Microsoft.Extensions.PlatformAbstractions": "1.0.0-*"
```
Comment (prafullbhosale) [inline src/Microsoft.Extensions.CodeGeneration.Utils/project.json]: Isn't `PlatformAbstractions` brought in by `"Microsoft.DotNet.ProjectModel": "1.0.0-*"`

File: src/Microsoft.Extensions.CodeGeneration.Core/project.json [hunk 2]
```diff
@@ -9,6 +9,7 @@
     "Microsoft.Extensions.CodeGeneration.Templating": "1.0.0-*",
     "Microsoft.Extensions.CommandLineUtils": "1.0.0-*",
     "Microsoft.Extensions.DependencyInjection": "1.0.0-*",
+    "Microsoft.Extensions.PlatformAbstractions": "1.0.0-*",
```
Comment (prafullbhosale) [inline src/Microsoft.Extensions.CodeGeneration.Core/project.json]: This project depends on `Utils` project below. So adding it there should be enough?

File: src/Microsoft.Extensions.CodeGeneration.Utils/project.json (hunk 1 above)
Comment (ryanbrandenburg) [inline src/Microsoft.Extensions.CodeGeneration.Utils/project.json]: Not anymore according to my reading of the code and @NTaylorMullen.

Comment (prafullbhosale) [inline src/Microsoft.Extensions.CodeGeneration.Utils/project.json]: Ah, I see, it was removed sometime last week.

File: src/Microsoft.Extensions.CodeGeneration.Core/project.json (hunk 2 above)
Comment (ryanbrandenburg) [inline src/Microsoft.Extensions.CodeGeneration.Core/project.json]: Oh I see, it's transitive from `Microsoft.Extensions.CodeGeneration.Templating`, I'll remove this one then.

**Comment by ryanbrandenburg** [comment]
:up::date:

**Comment by prafullbhosale** [comment]
:shipit:

**Pull request closed without merging**